code/modules/client/preferences.dm

```diff
@@ -1971,9 +1971,9 @@ GLOBAL_LIST_EMPTY(preferences_datums)
 				if("hotkeys")
 					hotkeys = !hotkeys
 					if(hotkeys)
-						winset(user, null, "input.focus=true input.background-color=[COLOR_INPUT_ENABLED]")
+						winset(src, null, "map.focus=true input.background-color=[COLOR_INPUT_DISABLED] mainwindow.macro=default")
```

Comment:
Intentional use of `src`? (src is a client's /datum/preferences in this context.)

Reply:
Nope. Didn't test this corner of things.